Research/graphql-project/app.py

```diff
@@ -3,17 +3,12 @@
 from flask_graphql import GraphQLView
 from schema import schema
 
-from mongoengine import connect
-from secrets import DB_URI
-
 app = Flask(__name__)
 
-
-
 app.add_url_rule("/graphql", view_func=GraphQLView.as_view('graphql', schema=schema, graphiql=True))
 
 
 if __name__ == '__main__':
-    connect(host=DB_URI)
-    # init_db()
+    #connect(host=DB_URI)
+    init_db()
     app.run(debug=True)
```

Comment:
Were these changes to the graphql research folder intentional?

Reply:
I think I intentionally changed it back to how it should be after accidentally changing it